doc/man1/openssl-x509.pod.in

```diff
@@ -645,6 +645,12 @@ extensions for a CA:
  openssl x509 -req -in careq.pem -extfile openssl.cnf -extensions v3_ca \
         -signkey key.pem -out cacert.pem
 
+Convert a SM2 certificate request into a self signed SM2 certificate:
+ 
+ openssl x509 -req -in careq.pem -extfile openssl.cnf -signkey key.pem \
```

Comment:
You need to add the newly added 'sm2-id' and 'sm2-hex-id' in 'SYNOPSIS' and leave descriptions of these options in 'Signing Options' I think.

Reply:
fixed